### ISSUE TITLE
CSV format data download incorrectly formatted.

### ISSUE THREAD

**Issue by rupertoverall** [genenetwork/genenetwork2#822]
A minor point, but the download buttons on trait pages (as in the attached screenshot ) offer CSV format and a helpful line of R code to read them in. The code will introduce errors though as the 6 header lines are not commented.

Best solution is to add '#' comments to the header lines (as suggested by the R and python code snippets).

<img width="667" alt="Screenshot 2023-12-12 at 14 51 41" src="https://github.com/genenetwork/genenetwork2/assets/35666678/571d3ca2-2f32-4f59-8798-3270a74798a5">

**Comment by ZulfiiaDitto** [comment]
@rupertoverall, 
As a suggestion you can utilize the following script 

import pandas as pd
trait = pd.read_csv('0026392353.csv', skiprows=7)

trait <- read.csv('10025905738.csv',skip = 7)

**Comment by rupertoverall** [comment]
@ZulfiiaDitto 
Yes - this what I have also been doing as a workaround. My comment was rather to the developers that this information should be included in the examples on the webpage. Commented-out lines are much friendlier to automated workflows.
